File: in-motion-server/Post/IMS.Post.Models/Dto/Outgoing/PostCommentReactionDto.cs
namespace IMS.Post.Models.Dto.Outgoing;

public class PostCommentReactionDto
{
    public string Id { get; set; }
    public PostAuthorDto Author { get; set; }

    public string Emoji { get; set; }
    
    public string PostCommentId { get; set; }
    
    public DateTime CreatedAt { get; set; }
}

File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/remote/dto/posts/comments/reactions/PostCommentReactionDto.kt
package com.inmotion.in_motion_android.data.remote.dto.posts.comments.reactions

import com.inmotion.in_motion_android.data.remote.dto.posts.PostAuthorDto

data class PostCommentReactionDto(
    val id: String,
    val author: PostAuthorDto,
    val emoji: String,
    val postCommentId: String,
    val createdAt: String
)
